.github/workflows/coverage_converter_publish_actions.yml

```
@@ -0,0 +1,148 @@
+name: Coverage Converter Publish Actions
+
+on:
+  pull_request:
+    types: [ opened, edited, synchronize ]
+    paths:
+      - 'metrics/coverage_converter/**'
+  push:
+    branches: master
+    paths:
+      - 'metrics/coverage_converter/**'
+
+jobs:
+  linux-actions:
+    name: Coverage Converter Linux Publish Actions
```

It seems like this action does not publish anything. The same for `macos-actions`. 
```suggestion
    name: Coverage Converter Linux Actions
```